.buildkite/deploy.pipeline.yml

```diff
@@ -27,46 +27,14 @@ docker_plugin: &docker_plugin_configuration
     propagate-environment: true
 
 steps:
-  - label: Extract platform resources
-    branches: master
-    command:
-      - .buildkite/scripts/extract_platform_resources.sh platform
-    artifact_paths:
-      - platform/*
-
-  - label: Extract platform resources
-    branches: devnet2
-    command:
-      - .buildkite/scripts/extract_platform_resources.sh platform
-    artifact_paths:
-      - platform/*
-    env:
-      BASE_VARIANT: devnet2
-
-  - label: Extract production-track platform resources
-    branches: master
-    command:
-      - .buildkite/scripts/extract_platform_resources.sh platform-prod
-    artifact_paths:
-      - platform-prod/*
-
-  - label: Extract production-track platform resources
-    branches: devnet2
-    command:
-      - .buildkite/scripts/extract_platform_resources.sh platform-prod latest-devnet2
-    artifact_paths:
-      - platform-prod/*
-
-  - wait
-
   - label: Get docker tag and save it as metadata for use later
     branches: master
     command: .buildkite/scripts/set_docker_tag_meta_data.sh
 
   - label: Build testing artifacts
-    branches: master devnet2
-    command:
-      - .buildkite/scripts/build_deployment_context.sh platform context.tar.gz
+    branches: master
+    command: # TODO: extract version.
```

Comment:
How will the version be extracted?

Reply:
once we can specify a vesrion we could parse cargo.toml? until then we can probably a file containing the version somewhere in the repo

Reply:
Something like a `.oasis_core_version` file in the repo root. Seems better than specifying the version in X places.

Reply:
Yeah sounds reasonable.